rudder-web/src/main/scala/com/normation/rudder/web/services/ReportDisplayer.scala

```diff
@@ -127,28 +132,117 @@ class ReportDisplayer(
     AnonFunc(ajaxCall)
   }
 
+
+  private[this] def displayIntro(report: NodeStatusReport): NodeSeq = {
```

Comment:
This is the only interest of the pull request :) - all the remaining is to be able to get the  RunAndConfigInfo in report here